lib/engine.py

```diff
@@ -1,10 +1,14 @@
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+from builtins import range
 import sys
 import os
 import getpass
 import zipfile
 import gzip
 import tarfile
-import urllib
+import urllib.request, urllib.parse, urllib.error
```

Comment:
I'm a little confused by this line (probably because I don't know much about Python 2/3 compatibility :smile:). We don't seem to use the last two imports and when we use `urllib.request` we call it as `urllib.request.urlopen`, which seems to work fine for me using just `import urllib`.

@goelakash - can you explain what's going on here to me? Thanks.